### PR TITLE
Add kefir v2.8.1 with git autoupdate

### PR DESCRIPTION
**Pull request by Piicksarn**
**`git repository : `** https://github.com/rpominov/kefir/tree/2.8.1
**`Watch : 51 /  Star : 714 /  Fork : 44`**

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

**Comment by Piicksarn** [comment]
@Amomo, would you mind to check out the pr for me?
Thank you~ :smiley:

File: ajax/libs/kefir/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "kefir",
+  "filename": "",
```
Comment (Amomo) [inline ajax/libs/kefir/package.json]: @Piicksarn you miss the filename~

File: ajax/libs/kefir/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "author": "Roman Pominov <rpominov@gmail.com>",
+  "homepage": "https://github.com/rpominov/kefir",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/rpominov/kefir.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/keen/keen-js.git",
```
Comment (Amomo) [inline ajax/libs/kefir/package.json]: your target is wrong!

**Comment by Piicksarn** [comment]
@Amomo, sorry for my careless!!!
would you mind to checkout the pr again?
Thank you a lot!

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look?
Thank you~ :-D

**Pull request merged**